hack/generator/pkg/astmodel/identifier_factory.go

```diff
@@ -11,12 +11,20 @@ import (
 	"unicode"
 )
 
+// \W is all non-word characters (https://golang.org/pkg/regexp/syntax/)
 var filterRegex = regexp.MustCompile(`[\W_]`)
 
+type Visibility string
+
+const (
+	Public   = Visibility("public")
+	Internal = Visibility("internal")
+)
+
```

Comment:
I'm conscious that _internal_ is really the .NET view on what to call that level of visibility; what's the Go nomenclature? perhaps _package_?

Reply:
exported / not exported, perhaps